django_datadog_logger/rest_framework/mixins.py

```diff
@@ -1,61 +0,0 @@
-import copy
-import logging
-
-from django.db import transaction
-from django.db.models import Manager, QuerySet
-from rest_framework.fields import ModelField
-
-logger = logging.getLogger(__name__)
-
-
-class ActionLogMixin:
```

Comment:
please could you explain why are you suggesting to delete this file from the library?